web/ui/module/codemirror-promql/.circleci/config.yml

```diff
@@ -0,0 +1,89 @@
+version: 2.1
```

Comment:
so actually it's not here for Prometheus. Indeed in Prometheus we will rely on the main configuration.
It will be used in the mirror repo for the release mainly

Reply:
Please remove them. The robot will add them to the mirror repository.